src/components/reusables/Fieldset.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+
+export interface FieldsetProps {
+  legend: string;
+  elements: Array<JSX.Element>;
+  labelClass?: string;
+  labelText?: string;
```

Comment:
I was going to say that `<label>`s should always be required when using any form elements, but you're using it differently. You don't need the `<label>` element inside the `<legend>` since the legend is already a label for the fieldset. This is confusing because you have two props that are both labels inside the legend, but I think the legend should have a more static label to describe the form instead of displaying information - if that makes sense. So something like "Library Stage" and the "current library stage" can be an element outside the label so it doesn't describe the following form. Missed this the first time.